formfactory/urls.py

```diff
@@ -10,4 +12,9 @@
         name="wizard-detail"
     ),
     url(r"^(?P<slug>[-\w]+)/$", FactoryFormView.as_view(), name="form-detail"),
+    url(
```

Comment:
No, you can't because of the way the csrf_exempt decorator works. I spent 3 hours trying all the combinations and I'm convinced there is no other way.